src/rust/ide/src/controller/module.rs

```diff
@@ -13,65 +13,27 @@ use ast;
 use ast::HasIdMap;
 use data::text::*;
 use double_representation as dr;
-use file_manager_client as fmc;
+use enso_protocol::file_manager as fmc;
+use fmc::API;
 use parser::Parser;
 
 
 
-// =======================
-// === Module Location ===
-// =======================
-
-/// Structure uniquely identifying module location in the project.
-/// Mappable to filesystem path.
-#[derive(Clone,CloneRef,Debug,Display,Eq,Hash,PartialEq)]
-pub struct Location(pub Rc<String>);
```

Comment:
To be considered if perhaps module Location should be only the "path" (as in relative segments without root id), not the whole `Path` structure. Root ID might be stored separately in the controller.
Not sure if this makes sense. Depends. Needs discussion.

Reply:
Idea rejected. Path must knows its root id, even if ephemeral.